src/graphql/schema/resolver-helpers.js

```diff
@@ -0,0 +1,34 @@
+export const mergeProps = (key, ...objects) => ({
+  key,
+  ...objects.reduce((arr, obj) => ({ ...arr, ...(obj[key] || obj(key)) }), {})
+});
+export const users = (arrayOfUserIDs, User) =>
+  User.find({ _id: { $in: arrayOfUserIDs } });
+
+export const voters = ({ voters }, _, { User }) => users(voters, User);
+
+export const creator = ({ creator }, _, { User }) => User.findOne(creator);
+
+export const buildSuggestionsObj = (dimension, authToken) => {
+  if (dimension.suggestions && dimension.suggestions.length) {
+    dimension.suggestions = dimension.suggestions.map(suggestion => ({
+      ...suggestion,
+      voters: [authToken],
```

Comment:
Why are we storing the authToken here? If someone - who has not created the trip - votes for that suggestion we have mixed data in the array. Would be consistent here and use the userID

Reply:
`buildSuggestionsObj()` is only used in the creation of a new trip where I think the logic to always make the logged-in user always is the creator of the suggestions is sound, right?